src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory_1.fhir.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Identical to `type2-first-mandatory.fhir.json`?

Reply:
That was a test case added by @grzegorz-robak but just renamed by me